### PR TITLE
feat(Partner Usage Reports): add new service to project

### PR DESCRIPTION
**Pull request by manukm-ibm**
## PR summary

Adding new partner API to fetch resource usage reports.
TRI: https://github.ibm.com/BSS/architecture/blob/master/design-docs/cosell/partner.md
Issue: https://github.ibm.com/BSS/metering-deliverables/issues/2063

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by manukm-ibm** [comment]
Integration test results
Screenshot 2024-01-23 at 12 52 40 PM
Examples result
Screenshot 2024-01-23 at 12 52 51 PM

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Service Name | Module Name | Service Class Name
 [IAM Policy Management](https://cloud.ibm.com/apidocs/iam-policy-management?code=python) | iam_policy_management_v1 | IamPolicyManagementV1
 [IBM Cloud Shell](https://cloud.ibm.com/apidocs/cloudshell?code=python) | ibm_cloud_shell_v1 | IbmCloudShellV1
 [Open Service Broker](https://cloud.ibm.com/apidocs/resource-controller/ibm-cloud-osb-api?code=python) | open_service_broker_v1 | OpenServiceBrokerV1
+[Partner Usage Reports](https://cloud.ibm.com/apidocs/partner-apis/resource-usage-reports?code=python) | partner_usage_reports_v1 | PartnerUsageReportsV1
```
Comment (padamstx) [inline README.md:63]: Need to make sure the correct URL is being specified across all 4 SDKs.

**Review by padamstx — COMMENTED** [review]
LGTM, just need to make sure the apidocs URL is correct.

File: README.md (hunk 1 above)
Comment (manukm-ibm) [reply README.md:64]: Made necessary updates

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**